src/geom_handlers/common_assessment.jl

```diff
@@ -196,16 +196,15 @@ function initial_search_rotation(
 end
 
 """
-    filter_intersecting_sites(res_df::DataFrame)::DataFrame
+    filter_sites(res_df::DataFrame)::DataFrame
```

Comment:
Rather than filtering sites, it's getting the `highest_scoring_site`?

Reply:
It's also filtering out any sites with a quality control flag attached to them. At the moment this is just sites with scores < 0.33 but we might need it to filter out problematic sites in the future?